frameworks/g-cloud-9/manifests/declaration.yml

```diff
@@ -137,8 +140,9 @@
   slug: application-accuracy
   editable: True
   prefill: False
-  description: >
-    You must confirm the accuracy of the information in this declaration and in your service descriptions. We might ask for proof after you’ve submitted your application.
+  description: |
+    You must confirm the accuracy of the information in this declaration and in your service descriptions. We might 
+    ask for proof after you’ve submitted your application.
```

Comment:
Do we want to force a newline in here? It should either be > with newlines or | on a single line, I think.